src/components/Timeline/TimelineWrapper.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import classnames from 'classnames'
+
+const propTypes = {
+  children: PropTypes.node
```

Comment:
Perhaps it's mandatory?

Reply:
Nope, I guess I wanted to write for className :/